src/pdn/src/pdn.tcl

```diff
@@ -254,6 +266,54 @@ proc define_pdn_grid {args} {
   }
 }
 
+sta::define_cmd_args "define_power_switch_cell" {[-name <name>] \
+                                                 [-control <control_name>] \
+                                                 [-acknowledge <acknowledge_name>] \
+                                                 [-power_switchable <power_switchable_net_name>] \
+                                                 [-power <power_net_name> ] \
+                                                 [-ground <ground_net_name>] }
+
+proc define_power_switch_cell {args} {
+  pdngen::check_design_state
+
+  sta::parse_key_args "define_power_switch_cell" args \
+    keys {-name -control -acknowledge -power_switchable -power -ground} 
+
+  if {[llength $args] > 0} {
+    utl::error PDN 1182 "Unexpected argument [lindex $args 0] for define_power_switch_cell command."
+  }
```

Comment:
Please mirror the other commands like `add_global_connection`:

```
sta::parse_key_args "add_global_connection" args \
    keys {-net -inst_pattern -pin_pattern} \
    flags {-power -ground -defer_connection}

  sta::check_argc_eq0 "add_global_connection" $args

  pdn::check_design_state "add_global_connection"
```
This way it's consistent with all the other calls.

Reply:
Done.